content/en/blog/2020/istiocon-2021/istiocon-2021.md

```diff
@@ -0,0 +1,16 @@
+# Join us for the first IstioCon in 2021!
+
+IstioCon 2021 will be the inaugural conference for Istio, the industry's most popular service mesh. In its inaugural year, IstioCon will be 100% virtual, connecting community members across the globe with Istio's ecosystem. This conference will take place at the end of February. 
```

Comment:
Which industry?

We could say "the world's" or similar, and refer to the CNCF 2020 survey which backs up the claim?

Reply:
I added a link to the survey. The wording was decided in collaboration with Srinath and Jacinda.